src/sources/redis/mod.rs

```diff
@@ -126,6 +128,10 @@ pub struct RedisSourceConfig {
     #[serde(default = "default_decoding")]
     #[derivative(Default(value = "default_decoding()"))]
     decoding: DeserializerConfig,
+
+    /// The namespace to use for logs. This overrides the global setting.
+    #[serde(default)]
```

Comment:
```suggestion
    /// The namespace to use for logs. This overrides the global setting.
    #[configurable(metadata(docs::hidden))]
    #[serde(default)]
```

From https://github.com/vectordotdev/vector/pull/15120